api/types/okta.go

```diff
@@ -385,10 +385,6 @@ func (o *OktaAssignmentV1) CheckAndSetDefaults() error {
 		return trace.BadParameter("user must not be empty")
 	}
 
-	if len(o.Spec.Targets) == 0 {
```

Comment:
This is OK to remove because all places where targets are relevant already do their own checks? Is that it?

Reply:
Yes,  empty targets should be evaluated as void operation  during assignment processing flow: 
 [cleanupTargets](https://github.com/gravitational/teleport.e/blob/f0d670881e1a1f472f7f4a6c1e2d5477f6e5ffa3/lib/okta/assignmentprocessor.go#L411) [processTargets](https://github.com/gravitational/teleport.e/blob/f0d670881e1a1f472f7f4a6c1e2d5477f6e5ffa3/lib/okta/assignmentprocessor.go#L359)